Assets/BossRoom/Scripts/Shared/ProfileManager.cs

```diff
@@ -16,32 +18,29 @@ public static class ProfileManager
 
         static string GetProfile()
         {
-#if UNITY_EDITOR
-
-            //The code below makes it possible for the clone instance to log in as a different user profile in Authentication service.
-            //This allows us to test services integration locally by utilising Parrelsync.
-            if (ClonesManager.IsClone())
+            var arguments = Environment.GetCommandLineArgs();
+            for (int i = 0; i < arguments.Length; i++)
             {
-                Debug.Log("This is a clone project.");
-                var customArguments = ClonesManager.GetArgument().Split(',');
+                if (arguments[i] == AuthProfileCommandLineArg)
+                {
+                    var profileId = arguments[i + 1];
+                    return profileId;
+                }
+            }
 
-                //second argument is our custom ID, but if it's not set we would just use some default.
+#if UNITY_EDITOR
 
-                var hardcodedProfileID = customArguments.Length > 1 ? customArguments[1] : "defaultCloneID";
+            // When running in the Editor and not a ParrelSync clone, make a unique ID
```

Comment:
Since this now works with ParrelSync clones too, we might want to change this comment